internal/app/app.go

```diff
@@ -127,7 +130,7 @@ func NewFileStorage(ctx context.Context, logger log.Logger) filestorage.Driver {
 // NewCalendar creates a new connection with the calendar service
 func NewCalendar(ctx context.Context, logger log.Logger) calendar.Calendar {
 	configuredCalendar := config.Env.Calendar
-	logger.Info(
+	logger.Debug(
```

Comment:
I would actually keep this one as Info, since it is just a one-line log, runs only once, in the beginning, and brings a relevant information (the configured calendar)